### PR TITLE
Fix galera HA

### PR DESCRIPTION
**Pull request by jhesketh**
Galera requires 3 nodes for HA. Now that mariadb is the default, update the jobs to use 3 controllers.

This also requires removal of drbd as that is locked to 2 nodes. Instead use nfs.

### CODE REVIEW

**Comment by jsuchome** [comment]
Looks correct to me, @skazi0 would you check the ironic jobs?

File: jenkins/ci.suse.de/templates/cloud-mkcloud-job-ha-compute-template.yaml [hunk 1]
```diff
@@ -22,8 +22,8 @@
             nodenumber=4
```
Comment (jsuchome) [inline jenkins/ci.suse.de/templates/cloud-mkcloud-job-ha-compute-template.yaml]: You need to bum this as well

**Comment by jsuchome** [comment]
I'm not sure about `nodenumberironicnode`, it was matching the number of controller nodes...

**Comment by skazi0** [comment]
@jsuchome `nodenumberironicnode` is independent of HA setup. The rest of ironic stuff should work properly or at least not worse than it was before :)

**Pull request merged**